step_size: 3   #maximum newton line search step size
max_iter: 100 #maxIterations
t_eps: 1e-8   #transformationEpsilon
res: 0.05 #Voxel side length for generating distributions
